app/assets/javascripts/discourse/app/templates/topic.hbs

```diff
@@ -69,6 +69,8 @@
             {{#if this.model.details.can_edit}}
               <a href {{action "editTopic"}} class="edit-topic" title={{i18n "edit"}}>{{d-icon "pencil-alt"}}</a>
             {{/if}}
+
+            <PluginOutlet @name="topic-title-icon" @tagName="" @connectorTagName="span" @args={{hash model=this.model}} />
```

Comment:
```suggestion
            <PluginOutlet @name="topic-title-suffix" @tagName="" @args={{hash model=this.model}} />
```

Two suggestions in the above: 
- removing the `@connectorTagName` means that the output is a little cleaner (and it's up to plugins/themes to choose exactly what the element type will be)
- these don't have to be icons, so might be better to call the outlet something else like `topic-title-suffix`

Reply:
I'm happy with those changes.